README.md

```diff
@@ -25,28 +25,243 @@ dependencies {
 ```
 
 ## Euphony is very easy to use
+## in Transmitter
+
+---
+
+### Generate sound waves and transmits though a microphone.
+
+1. **Create EuTxManager type object which turns data into sound waves.**
+- Java
 
-in Transmitter
 ```java
 EuTxManager mTxManager = new EuTxManager();
-mTxManager.euInitTransmit("Hello, Euphony"); // To generate acoustic data "Hello, Euphony"
-mTxManager.process(-1); // generate sound infinite.
 ```
 
-in Receiver
+- Kotlin
+
+```kotlin
+val mTxManager = EuTxManager()
+```
+
+2. **Use euInitTransmit method, enter the data you want to transmit.**
+- Java
+
+```java
+mTxManager.euInitTransmit("Hello, Euphony");
+```
+
+- Kotlin
+
+```kotlin
+mTxManager.euInitTransmit("Hello, Euphony")
+```
+
+3. **With process method, set how many times to repeat the sound(data). (-1 is infinite)**
+- Java
+
+```java
+mTxManager.process(-1);
+```
+
+- Kotlin
+
+```kotlin
+mTxManager.process(-1)
+```
+
+## in Receiver
+
+---
+
+### Part of receiving sound waves.
+
+> **Requires permission of RECORD_AUIDO**
+
+- Write in AndroidManifest.xml
+
+```java
+<uses-permission android:name="android.permission.RECORD_AUDIO" />
+```
+
+1. **Create EuRxManager object that receives sound waves as data**
+- Java
+
 ```java
 EuRxManager mRxManager = new EuRxManager();
+```
+
+- Kotlin
+
+```kotlin
+val mRxManager = EuRxManager()
+```
+
+2. **Listener settings that receive and process sound waves**
+- Java
+
+```java
 mRxManager.setAcousticSensor(new AcousticSensor() {
-@Override
-    public void notify(String letters) {
-        //when data is received
+		@Override
+		public void notify(String letters) {
+      //when data is received
     }
-});
+ });
+```
```

Comment:
I think this code block indent is little awkward. Would you update indent?

### as-is
```java
mRxManager.setAcousticSensor(new AcousticSensor() {
		@Override
		public void notify(String letters) {
      //when data is received
    }
 });
```

### to-be
```java
mRxManager.setAcousticSensor(new AcousticSensor() {
	@Override
	public void notify(String letters) {
		//when data is received
	}
});
```

Reply:
I'll edit this too!